src/triggers/ipfs.ts

```diff
@@ -22,3 +23,24 @@ export const unpinnedTrackContent: (cidField: string, limit?: number) => Trigger
     });
     return cids;
   };
+
+export const unpinnedProcessedArtworks = function (limit?: number): Trigger<undefined> {
```

Comment:
mostly copy paste from the above query. i could use more shared code, but just decided to copy paste incase changes need to be made, and i see you do that mostly as well.

Reply:
ye has kinda been refreshing just doing raw sql quick and skipping most ORM stuff tbh lol